### PR TITLE
Update troubleshoot-tcpip-port-exhaust.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2419
Added updated blog link and added note.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/troubleshoot-tcpip-port-exhaust.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ You may also see CLOSE_WAIT state connections in the same output, however CLOSE_
...
 >
->Netstat has been updated in Windows 10 with the addition of the **-Q** switch to show ports that have transitioned out of time wait as in the BOUND state.  An update for Windows 8.1 and Windows Server 2012R2 has been released that contains this functionality. The PowerShell cmdlet `Get-NetTCPConnection` in Windows 10 also shows these BOUND ports.
+>Netstat has been updated in Windows 10 with the addition of the **-Q** switch to show ports that have transitioned out of time wait as in the BOUND state.  An update for Windows 8.1 and Windows Server 2012R2 has been released that contains this functionality. The PowerShell cmdlet `Get-NetTCPConnection` in Windows 10 also shows these BOUND ports. Until 2016/10, netstat was inaccurate. Fixes for netstat were backported to 2012 R2. Network reporting tools, such as Netstat and PowerShell-based Get-NetTcpConnection don't report Transport Control Protocol (TCP) or User Datagram Protocol (UDP) port usage correctly since Windows Vista because some new TCP/IP features are introduced. This update brings some changes to Netstat.exe and Get-NetTcpConnection so that they can correctly report the TCP or UDP port usage in Windows Server 2012 R2.
```
Comment (nenonix) [inline windows/client-management/troubleshoot-tcpip-port-exhaust.md]:
```suggestion
>Netstat has been updated in Windows 10 with the addition of the **-Q** switch to show ports that have transitioned out of time wait as in the BOUND state.  An update for Windows 8.1 and Windows Server 2012 R2 has been released that contains this functionality. The PowerShell cmdlet `Get-NetTCPConnection` in Windows 10 also shows these BOUND ports.
>
>Until 10/2016, netstat was inaccurate. Fixes for netstat, back-ported to 2012 R2, allowed Netstat.exe and Get-NetTcpConnection to correctly report TCP or UDP port usage in Windows Server 2012 R2. See [Windows Server 2012 R2: Ephemeral ports hotfixes](https://support.microsoft.com/help/3123245/update-improves-port-exhaustion-identification-in-windows-server-2012) to learn more.
```

**Comment by e0i** [comment]
@kaushika-msft Would you review this PR based on issue #2419 ? Thank you.

**Comment by e0i** [comment]
@kaushika-msft @Justinha Would you review this PR based on issue #2419 ? Thank you.

**Pull request merged**